athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/data/SchemaBuilder.java

```diff
@@ -279,6 +280,14 @@ public SchemaBuilder addMetadata(String key, String value)
         return this;
     }
 
+    public SchemaBuilder addMetadata(String key, String value, Set<String> excludeKey)
```

Comment:
as a rule, every public method needs java doc and most private methods that are non-trivial do too. 

What is excludeKey's meaning?

Reply:
This is created to prevent the map of raw colum name to date format from being added to the schema metadata. instead we normalize the map using column name mapping and add that to the schema metadata.